src/content/en/updates/2017/04/native-mac-os-notifications.md

```diff
@@ -0,0 +1,145 @@
+and fixes a number of long standing bugs, such as Chrome not respecting the
+system Do Not Disturb setting.
+
+Below we'll look at the differences this change introduces to the existing
+API's.
+
+## Notification Center
+
+One of the benefits (and feature requests from users) of this change is that
+notifications will be displayed in OS X's notification center.
+
+<figure>
+  <img src="/web/updates/images/2017/04/macos-notifications/image00.png">
+  <figcaption>
+    Google Chrome Notifications will be displayed in the Mac OS X notification center.
+  </figcaption>
+</figure>
+
+## Differences
+### Icon Size and Positioning
+
+The appearance of icons will change. They'll be smaller in size and padding is
+applied. You may want to consider switching to a transparent background icon
+instead of a solid color to be aesthetically pleasing.
+
+<figure>
+  <img src="/web/updates/images/2017/04/macos-notifications/image01.png">
+  <figcaption>
+    Before and after of notification icons displayed by Chrome vs displayed by
+    Mac OS X.
+  </figcaption>
+</figure>
+
+### Action Icons
+
+Before this change action buttons and icons would be displayed in the
+notification. With the native notifications the action button icons will not be
+used and the user will need to hover over the notification and select the "More"
+button to see the available actions.
+
+<figure>
+  <img src="/web/updates/images/2017/04/macos-notifications/image02.png">
+  <figcaption>
+    Before and after of notification action buttons with icons displayed by
+    Chrome vs displayed by Mac OS X.
+  </figcaption>
+</figure>
+
+### Chrome Logo
+
+The Chrome logo will always be displayed and cannot be replaced or altered. This
+is a requirement for third party applications on Mac OS X.
+
+### Images
+
+The image option will no longer be supported on OS X. If you define an image
+property the notification will still be displayed, but it will ignore the image
+parameter (See example below).
+
+<figure>
+  <img src="/web/updates/images/2017/04/macos-notifications/image03.png">
+  <figcaption>
+    Before and after of notification image displayed by Chrome vs displayed by Mac OS X.
+  </figcaption>
+</figure>
+
+You can feature detect image support with the following code:
+
+```
+if ('image' in Notification.prototype) {  
+  // Image is supported.
+} else {  
+  // Image is NOT supported.
+}
+```
+
+## Chrome Extension Changes
+
+Chrome extensions have the concept of [notification
+templates](https://developer.chrome.com/apps/notifications#type-TemplateType)
+which will behave differently with this change.
+
+The image notification template will no longer show the image. You should ensure
+that images are supplemental and not required to be useful to your users.
+
+<figure>
+  <img src="/web/updates/images/2017/04/macos-notifications/image04.png">
+  <figcaption>
+    Before and after of example of the image template in the chrome.notification API.
```

Comment:
Maybe 'Before and after for image templates in the chrome.notification API. It's the phrase 'Before and after of example' that's striking me as awkward. Please address that throughout.